modules/xmpp/xmpp.js

```diff
@@ -553,4 +580,91 @@ export default class XMPP extends Listenable {
 
         return details;
     }
+
+    /**
+     * Notifies speaker stats component if available that we are the new
+     * dominant speaker in the conference.
+     * @param {String} roomJid - The room jid where the speaker event occurred.
+     */
+    sendDominantSpeakerEvent(roomJid) {
+        // no speaker stats component advertised
+        if (!this.speakerStatsComponentAddress || !roomJid) {
+            return;
+        }
+
+        const msg = $msg({ to: this.speakerStatsComponentAddress });
+
+        msg.c('speakerstats', {
+            xmlns: 'http://jitsi.org/jitmeet',
+            room: `${roomJid}` })
```

Comment:
Unnecessary template - you can use just `roomJid` directly.

Reply:
Fixed.